pkg/cmd/log.go

```diff
@@ -0,0 +1,223 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/rest"
+)
+
+type LogOptions struct {
+	ContainerName string
```

Comment:
Shouldn't we provide the `ContainerName` as a flag like in `kubectl logs` with the `-c`?